pkgs/development/python-modules/python-ndn/default.nix

```diff
@@ -4,33 +4,39 @@
 , buildPythonPackage
 , fetchFromGitHub
 , lark
+, poetry-core
+, poetry-dynamic-versioning
 , pycryptodomex
 , pygtrie
 , pytestCheckHook
+, pythonRelaxDepsHook
 , pythonOlder
 , setuptools
 }:
 
 buildPythonPackage rec {
   pname = "python-ndn";
-  version = "0.3-3";
+  version = "0.4.1";
 
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "named-data";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-XS71oIydyLYtx6OQGO5NrhjVSyZxnhufrZ1y/6TffPo=";
+    hash = "sha256-ArTP4LQu7VNjI/N13gMTc1SDiNmW5l4GdLYOk8JEfKg=";
   };
 
   disabled = pythonOlder "3.9";
```

Comment:
According to the changelog, the minimum required python version is 3.10 since version 0.4.0.

Reply:
Good catch